releasenotes/notes/add-systemd-support-for-suse-6029483da860fcfa.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Add SystemD support for SUSE
```

This could be confusing for users, we already do support "systemd" (as the service manager used by the Agent) on SUSE. Let's write:
```suggestion
    Add SystemD integration support for SUSE.
```
instead.